modules/cudf/src/data_frame.ts

```diff
@@ -876,12 +876,40 @@ export class DataFrame<T extends TypeMap = any> {
     const temp       = new Table({columns: this.select(subset)._accessor.columns});
     const series_map = {} as SeriesMap<T>;
     this._accessor.names.forEach((name, index) => {
-      if (this.get(name) instanceof Float32Series || this.get(name) instanceof Float64Series) {
+      if ([new Float32, new Float64].some((t) => this.get(name).type.compareTo(t))) {
         series_map[name] = Series.new(temp.getColumnByIndex(index).nans_to_nulls());
       } else {
         series_map[name] = Series.new(temp.getColumnByIndex(index));
       }
     });
     return new DataFrame(series_map);
   }
+
+  /**
+   * Creates a DataFrame of `BOOL8` Series where `true` indicates the value is null and
+   * `false` indicates the value is valid.
+   *
+   * @returns a DataFrame containing Series of 'BOOL8' where 'true' indicates the value is null
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new([0, null, 2]);
+   *  b: Series.new(['foo', 'bar', null]);
+   * });
+   *
+   * df.isNull()
+   * // return {
+   * //    a: [false, true, false],
+   * //    b: [false, false, true],
+   * // }
+   * ```
+   */
+  isNull(): DataFrame<{[P in keyof T]: Bool8}> {
+    return new DataFrame(new ColumnAccessor(this.names.reduce((cols, name) => {
+      cols[name] = this.get(name).isNull()._col;
+      return cols;
+    }, {} as ColumnsMap<{[P in keyof T]: Bool8}>)));
```

Comment:
I liked the style in https://github.com/rapidsai/node-rapids/pull/189. Does it work here too?

```suggestion
    return new DataFrame(this.names.reduce((cols, name) => ({
      ...cols,
      [name]: this.get(name).isNull()
    }), {} as SeriesMap<{[P in keyof T]: Bool8}>));
```

Reply:
Yeah, a bit neater.